### DISCUSSION TITLE
MITRE in R Studio

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#685] [Q&A] [answered]
Apakah ada referensi tentang penggunaan package MITRE di R Studio untuk Cybersecurity ?

Mohon bantuannya kak, terima kasih 🙏

**Comment by kinantytasyao** [comment]
Dear @yohanespm77 

Akan kami coba carikan terlebih dahulu untuk referensinya ya Pak

Terima kasih

**Reply by kinantytasyao** [reply, marked as answer]
Di R, MITRE dapat dilakukan dengan menggunakan package `mitre` atau `attckr`. Untuk lebih lengkapnya bisa coba dicek pada documentation berikut

- https://www.rdocumentation.org/packages/mitre/versions/0.5.2
- https://rdrr.io/github/hrbrmstr/attckr/

**Reply by yohanespm77** [reply]
Baik, terima kasih kak
